packages/jest-worker/src/workers/processChild.ts

```diff
@@ -97,7 +99,21 @@ function reportSuccess(result: unknown) {
     throw new Error('Child can only be used on a forked process');
   }
 
-  process.send([PARENT_MESSAGE_OK, result]);
+  try {
+    process.send([PARENT_MESSAGE_OK, result]);
+  } catch (error: unknown) {
```

Comment:
```suggestion
  } catch (error) {
```

it's `unknown` by default

Reply:
Thanks.
Fix https://github.com/jestjs/jest/pull/15191/commits/bdafa9150e6d25674b30d8d4de88b7724003a9b6